### DISCUSSION TITLE
Does anyone has a power bi model template for this connector

### DISCUSSION THREAD

**Discussion by mitchonfire** [migueesc123/PowerBIRESTAPI#151] [General]
Hi guys,

Does anyone know of a dataset model template which pulls in all the tables from this API connector and relates them in one semantic DAX/Tabular model? (a .pbit file)

I have been working on one for months but it is not perfect, wondering if there are any others out there?

**Comment by migueesc123** [comment]
I don't have one, but I'll leave this open in case anyone has anything.

Anyone is welcomed to contribute to this repo.

**Comment by Dranrew** [comment]
If someone has a .pbit file it would indeed be very helpful!
